### PR TITLE
Replace delete with remove certificate in RAT profile (EXPOSUREAPP-10976)

### PR DESCRIPTION
**Pull request by SamuraiKek**
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10976)

Replacing occurences of "delete profile" with "remove profile" for RAT profile.

### CODE REVIEW

**Review by BMItr — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values-de/antigen_strings.xml [hunk 1]
```diff
@@ -188,17 +188,17 @@
     <!-- XBUT: Create RAT Qr code profile next button-->
     <string name="rat_qr_code_profile_next_button">Weiter</string>
     <!-- XTXT: Create RAT Qr code profile delete item-->
-    <string name="rat_qr_code_profile_delete_item">Schnelltest-Profil löschen</string>
+    <string name="rat_qr_code_profile_delete_item">Schnelltest-Profil entfernen</string>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values-de/antigen_strings.xml:191]: I think we changed every `entfernen` into `löschen` intentionally ?

Comment (mtwalli) [reply Corona-Warn-App/src/main/res/values-de/antigen_strings.xml:191]: See https://github.com/corona-warn-app/cwa-app-android/pull/4357

Comment (CV113) [reply Corona-Warn-App/src/main/res/values-de/antigen_strings.xml:191]: HI, I thought that too... @davidlindercwa @hermesmar  Could you please confirm!?

**Review by CV113 — COMMENTED** [review]
See comment

**Comment by mtwalli** [comment]
Ticket is obsolete

**Pull request closed without merging**